devnet/vitosha-1/currencies.json

```diff
@@ -121,6 +122,31 @@
                             "ticker": "ATOM",
                             "decimal_digits": "6"
                         }
+                    },
+                    "NLS": {
+                        "ibc": {
+                            "network": "OSMOSIS",
+                            "currency": "NLS"
+                        }
+                    },
+                    "USDC": {
+                        "ibc": {
+                            "network": "OSMOSIS",
+                            "currency": "USDC_AXELAR"
+                        }
+                    },
+
+                    "OSMO": {
+                        "ibc": {
+                            "network": "OSMOSIS",
+                            "currency": "OSMO"
+                        }
+                    },
+                    "AKT": {
+                        "ibc": {
+                            "network": "OSMOSIS",
+                            "currency": "AKT"
+                        }
                     }
```

Comment:
Why do we need all of those assets here if we're forwarding through the Hub? These won't and should not be on the Hub. Only ATOM needs to be forwarded from the Hub.